parser.py

```diff
@@ -1,17 +1,30 @@
 #!/bin/env python
 import argparse
 import pprint
+import re
+
 import grammar
 import visitor
 
+class PreProcessor:
+    @staticmethod
+    def remove_comments(program: str):
```

Comment:
In my head this would look something like this:

```suggestion
    def preprocess(self, program: str):
```